src/components/checkbox/checkbox.test.js

```diff
@@ -415,29 +409,23 @@ context("Testing Checkbox component", () => {
           <CheckboxGroupComponent error="Error has occurred" />
         );
 
-        checkboxGroupIcon()
-          .should("have.attr", "aria-label", "Error has occurred")
-          .should("have.attr", "data-element", "error");
+        checkboxGroupIcon().should("have.attr", "data-element", "error");
```

Comment:
Similar to the single checkbox tests

```suggestion
        checkboxGroupIcon().should("have.attr", "data-element", "error");
        tooltipPreview().should("have.text", "Error has occurred");
```